### PR TITLE
Update all packages to latest major version (except ethers)

### PR DESCRIPTION
**Pull request by 0xemc**
## Description

Updates all package version to their latest major version except for Ethers.js due to migration work required.

Some minor work needed for Lingui macro

### CODE REVIEW

File: app/components/views/Offset/index.tsx [hunk 1]
```diff
@@ -599,9 +599,8 @@ export const Offset = (props: Props) => {
     .map((tkn) => ({
       ...tokenInfo[tkn],
       description: (function () {
-        if (isLoading) return <Trans id="shared.loading" />;
+        if (isLoading) return <Trans id="shared.loading">Loading...</Trans>;
         if (!props.isConnected || !Number(balances?.[tkn])) return "0";
-        return Number(balances?.[tkn]).toFixed(2);
```
Comment (ladytrekker) [inline app/components/views/Offset/index.tsx:604]: Why is this line removed here?

File: app/components/views/Stake/index.tsx [hunk 2]
```diff
@@ -349,21 +349,16 @@ export const Stake = (props: Props) => {
               </button>
             </div>
             <div className={styles.stakeInput}>
-              <Trans
-                id={inputPlaceholderMessage[view].id}
-                render={({ translation }) => (
```
Comment (ladytrekker) [inline app/components/views/Stake/index.tsx:354]: Did you check if the language switch still works here?
I remember that this render function was needed back then as the component did not re-render when the user switches the language and the placeholder remains in the former language

File: app/components/views/Wrap/index.tsx [hunk 3]
```diff
@@ -326,18 +326,13 @@ export const Wrap: FC<Props> = (props) => {
               </button>
             </div>
             <div className={styles.stakeInput}>
-              <Trans
-                id={inputPlaceholderMessage[view].id}
-                render={({ translation }) => (
```
Comment (ladytrekker) [inline app/components/views/Wrap/index.tsx:331]: Same here.

File: app/pages/_app.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ function MyApp({ Component, pageProps }: AppProps) {
   return (
     <>
       <StyledEngineProvider injectFirst>
-        <I18nProvider i18n={i18n} forceRenderOnLocaleChange={false}>
```
Comment (ladytrekker) [inline app/pages/_app.tsx:17]: Aha, maybe this check here is enough? see comments above

**Review by ladytrekker — COMMENTED** [review]
Some comments below.

I will test this PR myself again too next week.

File: app/components/views/Offset/index.tsx (hunk 1 above)
Comment (0xemc) [reply app/components/views/Offset/index.tsx:604]: Great pickup, I think it was from auto formatting 🤦‍♂️

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
Pls document any intentional code changes, sounds like they were unintentional?

I just merged the API pr so you might need to take a second pass at this 😅
